webapp/tests/test_tags.py

```diff
@@ -550,15 +559,15 @@ def test_tag_views(self):
     response = self.client.post(url + '/delSeries', {'path': 'test.a;blah=blah;hello=lion'})
     self.assertEqual(response.status_code, 200)
     self.assertEqual(response['Content-Type'], 'application/json')
-    self.assertEqual(response.content, json.dumps(expected))
+    self.assertEqual(response.content, json_bytes(expected, indent=2, sort_keys=True))
 
     # find nonexistent series
     expected = []
 
     response = self.client.get(url + '/findSeries?expr=name=test.a&expr=hello=tiger&expr=blah=blah')
     self.assertEqual(response.status_code, 200)
     self.assertEqual(response['Content-Type'], 'application/json')
-    self.assertEqual(response.content, json.dumps(expected, indent=2, sort_keys=True))
+    self.assertEqual(response.content, json_bytes(expected, indent=2, sort_keys=True))
```

Comment:
Technically since the last 4 tests don't pass `pretty=1` we should be expecting non-indented json (right now they pass because the indented json is only a single line).